pages/hub/apps/guides/custom-markets.mdx

```diff
@@ -0,0 +1,51 @@
+import { Callout } from 'components'
+
+The Azuro Protocol also offers markets for world events, such as the "2024 Presidential Election USA." 
```

Comment:
Apart from the predominant sports markets, Azuro now allows to integrate political and social markets, starting with the "2024 Presidential Election USA" market.

Reply:
ed713387326b0f5d2cc3aa32c1ea95d8f69953b4